### DISCUSSION TITLE
Soft assignment

### DISCUSSION THREAD

**Discussion by mattiasmar** [introlab/rtabmap#1101] [General]
Is soft assignment of KP features supported by rtabmap?
(As in: H. J´egou, M. Douze, and C. Schmid. Improving bag-of-features for large scale image search. IJCV , 2010.)

**Comment by matlabbe** [comment]
Soft assignment seems described more explicitly in that paper https://ieeexplore.ieee.org/document/4587635

but no, feature quantization is done using a basic Kd tree (kd forest) approximate NN.
